build.gradle

```diff
@@ -8,7 +8,7 @@ apply plugin: 'idea'
 apply plugin: 'com.jfrog.bintray'
 apply plugin: 'maven'
 
-version = '1.0.1'
+version = '1.0.2'
```

Comment:
:package: I don't know if what I have done here is correct. I could not find EmailValidator4J 1.0.2 in the public Maven repos. I thought it might be because the POM here was showing 1.0.1 still.